accounts-db/src/accounts_hash.rs

```diff
@@ -1344,6 +1363,7 @@ pub mod tests {
                 file2.write(hash);
                 combined.push(*hash);
             });
+            file2.release_writer();
```

Comment:
I could see this as being helpful if the # buf writers is a problem somehow.
Also, we could do something to limit the # of these parallel calls that are started and allowed to excecute. So a fewer number (64?) is the max that could have the buf writer open at a time.

Reply:
yeah. I think so.